### ISSUE TITLE
Modal mode indicator shown twice

### ISSUE THREAD

**Issue by farzadmf** [whame/tmux-modal#4]
I'm using [powerline](https://github.com/powerline/powerline) for my status bar, and it happens a lot that I see two equal signs in the status bar when I enable the modal mode. If I restart my tmux session, it goes back to only one.

What I see now:
![image](https://user-images.githubusercontent.com/5768792/154737123-e87923ad-402b-492c-add7-ed27061e0883.png)

What I see when I end my tmux session and start a new one:
![image](https://user-images.githubusercontent.com/5768792/154737337-e2007a03-8905-40cd-8e5b-af63389115fc.png)

**Comment by farzadmf** [comment]
I _think_ I was able to reproduce by using `tpm`' s `U`update function.

Basically:
- I open a `tmux` session -> doing `M-m` shows a single `=` sign
- I do `prefix + U` > `all` > `ENTER` to force a reload from `tpm`
- Now I have 2 `=` signs

**Comment by whame** [comment]
I just tried to reproduce with your steps, but I couldn't trigger it. Can you trigger this consistently with your reproducing steps?

I wonder if something else in your config causes this. Would it be possible for instance to comment out any other plugins and see if it still happens?

**Comment by farzadmf** [comment]
OK, good to know, TBH, I was a bit hesitent to create the issue as I thought it might have something to do with my setup.

I have a bunch of plugins installed, so I'll try to see if something is clashing with other plugins

**Comment by whame** [comment]
The output of 

```
tmux show-options -g -v status-left
```

might be helpful here as well. Save the output after a fresh session, and then when you trigger it, run the command again and paste the output so we can compare.

**Comment by whame** [comment]
I figured that we could be more robust if we only set the status bar once. It is better to not trust the environment to only load tmux-modal once. I just pushed a fix, please update and try it. We should be able to close this issue. Thank you for reporting this!

**Comment by farzadmf** [comment]
Nice! thank you for being so fast at looking at the issues.

Updated and it seems like the issue is fixed. I noticed that, for some reason, I wasn't able to get out of modal mode (or maybe I was out of it but the `=` was still there), but I haven't been able to reproduce.

I'll create a new issue if I see that happening, and gonna close this one

Again, well done with the plugin and the fast support to fix issues. Keep up the good work
